pkg/metainfo/kvmetainfo/buckets.go

```diff
@@ -48,6 +49,10 @@ func (db *Project) CreateBucket(ctx context.Context, bucketName string, info *st
 		info.SegmentsSize = db.segmentsSize
 	}
 
+	if err := validateBlockSize(info.RedundancyScheme, info.EncryptionParameters.BlockSize); err != nil {
```

Comment:
Just curious on the reason why we moved this function into `kvmetainfo`. More organized code?

Reply:
Good question. I guess two reasons: 
1) `validateBlockSize()` was happening in two different packages and I wondered if I could move it to a shared package
2) I realized that the flow of data as it relates to metainfo goes like this:
```
libuplink -> pkg/metainfo/kvmetainfo -> pkg/storage -> network layer/grpc -> SA metainfo endpoint -> SA pointerDB
```
Previously the `validateBlockSize` check was happening in two places:
1. `lib/uplink` when we created a bucket and 
2. `pkg/metainfo/kvmetainfo` when we created an object

This meant that `validateBlockSize` was occurring in two different layers, the highest layer `lib/uplink` and the next layer down in `pkg/metainfo/kvmetainfo`. So I figured we could move the check to the same layer `pkg/metainfo/kvmetainfo`.